tools/font-subset/BUILD.gn

```diff
@@ -41,7 +41,7 @@ generated_file("_font-subset-license") {
 }
 
 zip_bundle("font-subset") {
-  output = "font-subset.zip"
+  output = "${full_platform_name}/font-subset.zip"
```

Comment:
I think this could have been done with a soft transition without breaking the tree by having two copies of this rule, one with the old output and one with the new output?